docs/docs/features/combos.md

```diff
@@ -28,7 +27,7 @@ Combos configured in your `.keymap` file, but are separate from the `keymap` nod
 - The `compatible` property should always be `"zmk,combos"` for combos.
 - `timeout-ms` is the number of milliseconds that all keys of the combo must be pressed.
 - `key-positions` is an array of key positions. See the info section below about how to figure out the positions on your board.
-- `layers = <0 1...>` will allow limiting a combo to specific layers. this is an _optional_ parameter and defaults to `-1` which is global scope.
+- `layers = <0 1...>` will allow limiting a combo to specific layers. this is an _optional_ parameter, when omitted it defaults to `-1` which is global scope.
```

Comment:
Please capitalize `This` and don't mention the "defaults to -1" part. Simply "it defaults to global scope" is better because otherwise people will try to use the -1 value anyway :)

Reply:
Agreed, better to keep `-1` under the hood then. 
Fixed